### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by RuidsCosta**
needs adjustment on whidth

### CODE REVIEW

File: components/NavBar.js [hunk 1]
```diff
@@ -1,56 +1,137 @@
...
+              justify="flex-start"
+              alignItems="center"
+              spacing={2}
+            >
+              <Grid item><Button color="inherit">Contactos</Button></Grid>
+              <Grid item><Button color="inherit">Preços</Button></Grid>
+              <Grid item><Button color="inherit">Atividades</Button></Grid>
+              <Grid item><Button color="inherit">Notícias</Button></Grid>
+              <Grid item><Button color="inherit">Equipa</Button></Grid>
+              <Grid item><Button color="inherit">Sobre</Button></Grid>
```
Comment (nelsonmestevao) [inline components/NavBar.js]: Should just be a map :point_up:

File: components/NavBar.js [hunk 2]
```diff
@@ -1,56 +1,137 @@
...
-  </StyledTypography>
-  );
+const options = [
+  'Contactos',
+  'Preços',
+  'Atividades',
+  'Notícias',
+  'Equipa',
+  'Sobre',
+];
```
Comment (nelsonmestevao) [inline components/NavBar.js]: Could you add a link for each button? And probably move it to a `.json`.

File: components/NavBar.js [hunk 3]
```diff
@@ -1,56 +1,137 @@
...
-        </Grid>
-      </Grid>
-    </StyledToolBar>
-  </StyledBox>
-  );
 
-Entry.propTypes = {
-  name: PropTypes.string.isRequired,
-};
+export default function ButtonAppBar() {
```
Comment (nelsonmestevao) [inline components/NavBar.js]: It should probably be split into multiple components :pray:

File: components/NavBar.js (hunk 1 above)
Comment (miguelbrandao) [reply components/NavBar.js]: Done in the refactor

File: components/NavBar.js (hunk 2 above)
Comment (miguelbrandao) [reply components/NavBar.js]: Done in the refactor

File: components/NavBar.js (hunk 3 above)
Comment (miguelbrandao) [reply components/NavBar.js]: I'll take a look :+1:

**Comment by miguelbrandao** [comment]
There is still some stuff left to do:
 - adding a background when scrolling
 - adding some style to the buttons when hovering, like and underline or a background

I will add links later.

File: components/navbar/NavBar.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+            <Grid
+              container
+              direction="row-reverse"
+              justify="flex-start"
+              alignItems="center"
+              spacing={2}
+            >
+              {global.navbar.pages
+                .map((page) => (
+                  // TODO: add links. Because the NextJS router assumes links in Link are internal,
```
Comment (herulume) [inline components/navbar/NavBar.js]: Add the missing code or remove this please.

File: components/navbar/NavBar.js [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+import AppBar from '@material-ui/core/AppBar';
+import Toolbar from '@material-ui/core/Toolbar';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import Grid from '@material-ui/core/Grid';
+import useMediaQuery from '@material-ui/core/useMediaQuery';
+import global from '../../static/global.json';
+import BurgerButton from './BurgerButton';
+
+const useStyles = makeStyles((theme) => ({
```
Comment (herulume) [inline components/navbar/NavBar.js]: We are trying to use Styled Components, refactor this to make everything consistent.

File: components/navbar/NavBar.js (hunk 4 above)
Comment (miguelbrandao) [reply components/navbar/NavBar.js]: This code will be added as a standalone component very, very soon and the comment is there as a reminder to change that bit when it is implemented. I don't think it's an issue to let it stay in _dev_ for a few days.

File: components/navbar/NavBar.js (hunk 5 above)
Comment (miguelbrandao) [reply components/navbar/NavBar.js]: Done!

**Pull request merged**